weeks/week05/modules.md

```diff
@@ -1,3 +1,150 @@
 # Section 1: Modules
 
-Coming soon.
+Python modules and Python packages (which will be covered in the next section), two mechanisms that facilitate modular programming.
+
+According to modular programming concept, large tasks are broken into smaller manageable subtasks or modules. Modularized code has the following benefits:
+
+- Readability
+- Maintainability
+- Reusability
+- Scoping
+- Manageability
+
+In this section will show how to create and import a module in Python.
```

Comment:
I think this phrase should be enhanced for clarity. Maybe remove "In" from the beginning.